book_source/03_intermediate_users_guide/01_pecan_xml.Rmd

```diff
@@ -1,195 +1,487 @@
+  
+* **edin** : [required] template used to write ED2IN file
+* **veg** : **OBSOLETE** [required] location of VEG database, now part of `<run><inputs>`
+* **soil** : **OBSOLETE** [required] location of soild database, now part of `<run><inputs>`
+* **psscss** : **OBSOLETE** [required] location of site inforation, now part of `<run><inputs>`. Should be specified as `<pss>`, `<css>` and `<site>`.
+* **inputs** : **OBSOLETE** [required] location of additional input files (e.g. data assimilation data), now part of `<run><inputs>`. Should be specified as `<lu>` and `<thsums>`.
+
+### `run`: Run Setup {#xml-run}
 
+This section provides detailed configuration for the model run, including the site and time period for the simulation and what input files will be used.
 
-## Meta Analysis
+```xml
+  <run>
+    <site>
+      <id>1000000098</id>
+      <met.start>2004/01/01</met.start>
+      <met.end>2004/12/31</met.end>
+    </site>
+    <inputs>
+      <met>
+        <source>CRUNCEP</source>
+        <output>SIPNET</output>
+      </met>
+    </inputs>
+    <start.date>2004/01/01</start.date>
+    <end.date>2004/12/31</end.date>
+  </run>
+```
 
-#### functions:
+#### `site`: Where to run the model {#xml-run-site}
 
-`run.meta.analysis()`
+This contains the following tags:
 
-#### tags
+- `id` -- This is the numeric ID of the site in Bety (table `sites`, column `id`). PEcAn can automatically fill in other relevant information for the site (e.g. `name`, `lat`, `lon`) using the site ID, so those fields are optional if ID is provided.
+- `name` -- The name of the site, as a string.
+- `lat`, `lon` -- The latitude and longitude coordinates of the site, as decimals.
+- `met.start`, `met.end` -- ???
+
+#### `inputs`: Model inputs {#xml-run-inputs}
+
+Models require several different types of inputs to run.
+Exact requirements differ from model to model, but common inputs include meteorological/climate drivers, site initial conditions (e.g. vegetation composition, carbon pools), and land use drivers.
+
+In general, all inputs should have the following tags:
+
+* `id`:  Numeric ID of the input in Bety (table `inputs`, column `id`). If not specified, PEcAn will try to figure this out based on the `source` tag (described below).
+* `path`: The file path of the input. Usually, PEcAn will set this automatically based on the `id` (which, in turn, is determined from the `source`). However, this can be set manually for data that PEcAn does not know about (e.g. data that you have processed yourself and have not registered with Bety).
+* `source`: The input data type. This tag name needs to match the names in the corresponding conversion functions. If you are using PEcAn's automatic input processing, this is the only field you need to set. However, this field is ignored if `id` and/or `path` are provided.
```

Comment:
Seems like a natural place to link to info about `the corresponding conversion functions`

Also, at a higher level, this whole section on inputs seems to missing all the new information about how ensemble inputs are specified.